build/xwalk.gni

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file contains Crosswalk-feature-related build flags. Users should
+# import it into their build folder by run:
+#   echo 'import("//xwalk/build/xwalk.gni")' > out/your_folder/args.gn'
+# Then run:
+#   gn gen out/your_folder
+
+# Set it false when building content_shell.
+declare_args() {
+  is_build_crosswalk = true
+}
+
+if (is_build_crosswalk) {
+  target_os = "linux"
+  target_cpu = "x64"
+  is_debug = false
+
+  # From src/build/config/features.gni
+  # TODO: Enable nacl for Linux.
+  enable_nacl = false
+
+  # From src/build/config/features.gni
+  enable_plugins = true
+
+  # From src/build/config/features.gni
+  enable_webvr = false
+
+  # From src/third_party/widevine/cdm/widevine.gni
+  enable_widevine = false
+
+  # From src/build/config/compiler/BUILD.gn
+  # If we want stack unwind support for backtrace()
+  exclude_unwind_tables = false
+
+  # From src/third_party/ffmpeg/ffmpeg_generated.gni
+  ffmpeg_branding = "Chromium"
```

Comment:
For variables whose value differs from the upstream default only for the non-Linux case (e.g. in gyp we only change `ffmpeg_branding` on Android), I think it makes more sense not to include them in this commit. It makes it much easier to understand where they came from if they are added with the rest of the code to support Android, Windows and other targets later.

This applies to other variables, such as `use_rssdk`, `exclude_unwind_tables` and `v8_use_external_startup_data`.